File: src/java/WebNinja/src/test/java/webninja/selenuim/IssuePage.java
package webninja.selenuim;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import webninja.UserRepository;

public class IssuePage {
    private final WebDriver webDriver;
    private final CodeTrack codeTrack;
    private final UserRepository userRepository;

    public IssuePage(WebDriver webDriver, CodeTrack codeTrack, UserRepository userRepository) {
        this.webDriver = webDriver;
        this.codeTrack = codeTrack;
        this.userRepository = userRepository;
    }


    public IssueForm startEdit() {
        webDriver.findElement(By.xpath("//input[@type='submit']")).click();
        return new IssueForm(webDriver, codeTrack, userRepository);
    }


}

File: src/dotnet/WebNinja/WebNinja/selenium/IssuePage.cs
using OpenQA.Selenium;
using WebNinja.webninja;

namespace WebNinja.selenium
{
    public class IssuePage : PageObject
    {
        private readonly UserRepository _repository;

        public IssuePage(IWebDriver driver, UserRepository repository) : base(driver)
        {
            _repository = repository;
        }

        public IssueForm StartEdit()
        {
            Driver.FindElement(By.XPath("//input[@type='submit']")).Click();
            return new IssueForm(Driver, _repository);
        }
    }
}
